test/api_integration/config.js

```diff
@@ -22,7 +22,7 @@ export default async function ({ readConfigFile }) {
     },
     esTestCluster: {
       ...functionalConfig.get('esTestCluster'),
-      serverArgs: ['xpack.security.enabled=false'],
+      serverArgs: ['xpack.security.enabled=false', 'search.check_ccs_compatibility=true'],
```

Comment:
I was thinking you could just put that setting here and it would cover everything;
https://github.com/elastic/kibana/blob/main/test/common/config.js#L24